Symfony/CS/FixerInterface.php

```diff
@@ -75,6 +75,8 @@ public function getPriority();
     /**
      * Returns true if the file is supported by this fixer.
      *
+     * @param \SplFileInfo $file A \SplFileInfo instance
```

Comment:
I tend to disagree, IMO the docs should be complete or left out.
@keradus what do you think?